packages/@angular/cli/utilities/find-parent-module.ts

```diff
@@ -2,11 +2,13 @@ import * as fs from 'fs';
 import * as path from 'path';
 const SilentError = require('silent-error');
 
-export default function findParentModule(project: any, currentDir: string): string {
-  const sourceRoot = path.join(project.root, project.ngConfig.apps[0].root, 'app');
+export default function findParentModule
```

Comment:
Put as many arguments on the same line as function declaration, then indent 4 (or left align with first argument) the rest.